contract-ui/tabs/settings/components/metadata.tsx

```diff
@@ -154,22 +160,22 @@ export const SettingsMetadata = <TContract extends PotentialContractInstance>({
           </Flex>
         </Flex>
 
-        {/*         <AdminOnly contract={contract}> */}
```

Comment:
question:
doesn't `AdminOnly` just make the same `useIsAdmin()` check underneath? if so why do we need to change this?

Reply:
just checked and it doesn't, but we could *make* it use `useIsAdmin()` underneath and then we can change it in a better place?